models/sources/models.sql

```diff
@@ -14,6 +14,8 @@ select
     cast(null as {{ type_string() }}) as package_name,
     cast(null as {{ type_string() }}) as path,
     cast(null as {{ type_string() }}) as checksum,
-    cast(null as {{ type_string() }}) as materialization
+    cast(null as {{ type_string() }}) as materialization,
+    cast(null as {{ type_array() }}) as tags,
+    cast(null as {{ type_json() }}) as meta
```

Comment:
@NiallRees I'm not sure what the linter is complaining about, I can't see any trailing whitespace on those lines!

Reply:
There is a bug with SQLFluff atm for L001 - https://github.com/sqlfluff/sqlfluff/issues/3851 - We managed to fix by reverting to 1.2.1.